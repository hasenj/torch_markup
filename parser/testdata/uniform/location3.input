//to:-print-mode=printoffsets
__a__
