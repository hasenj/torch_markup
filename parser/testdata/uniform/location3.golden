0-5: Container()(
	0-5: Leaf(T)(
		0-5: Container()(
			0-5: Uniform(MA)(
				2-3: Container()(
					2-3: Text(MT)(
						a
					)
				)
			)
		)
	)
)